langtest/augmentation/__init__.py

```diff
@@ -93,15 +93,18 @@ def __init__(
             with open(self.config) as fread:
                 self.config = yaml.safe_load(fread)
 
-    def fix(self, input_path: str, output_path, inplace: bool = False):
+    def fix(self, input_path: str, output_path, inplace: Union[bool, str] = False):
         """
         Applies perturbations to the input data based on the recommendations from harness reports.
 
         Args:
             input_path (str): The path to the input data file.
             output_path (str): The path to save the augmented data file.
-            inplace (bool, optional): If True, the list of samples is modified in place.
-                                      Otherwise, a new samples are add to input data. Defaults to False.
+            inplace (Union[bool, str], optional): Determines how the samples are modified or exported.
+                                                If True, the list of samples is modified in place.
+                                                If False, new samples are added to the input data.
```

Comment:
Instead of having a boolean or a string can we make it so that we always have a string which can take the following values: 'inplace', 'add', 'transformed'?

Maybe rename the parameter `inplace` -> `export_mode`